demo/config.ru

```diff
@@ -9,17 +9,13 @@ def generate_html_with(env)
   pix         = QrcodePixRuby::Payload.new
 
   unless qrcode_data.empty?
-    pix.pix_key        = qrcode_data['pix_key']           if !qrcode_data['pix_key'].nil?        && !qrcode_data['pix_key'].empty?
```

Comment:
@marciojg Maybe we can write this snippet a little bit different, in order to keep `repeatable` working in form (if we delete it from hash, the respective form field appears empty for user 😞):

```ruby
qrcode_data.keys.each do |key|
  v = qrcode_data[key]
  pix.public_send("#{key}=", key == 'repeatable' ? v == 't' : v) if !v.nil? && !v.empty?
end
```

Reply:
Ok. I will run some tests with the Docker file that you create!

Reply:
After this change, we are ready for merge 🎉  ✅

Reply:
@pedrofurtado I haven't been able to do it yet.

Soon I will do it. :)